README.md

```diff
@@ -37,6 +37,7 @@ This started as a very lightweight, single page html file, and we've tried very
 To run a development server that will auto-reload on save, run this command from the project directory:
```

Comment:
Minor: these commands

Reply:
Moved the npm install to it's own instruction since you don't have to do that every time you run the dev server.